lib/read/lib/Cardano/Wallet/Read/Block/Block.hs

```diff
@@ -56,6 +56,8 @@ import qualified Ouroboros.Consensus.Shelley.Ledger as O
 type ConsensusBlock = O.CardanoBlock O.StandardCrypto
 
 -- Family of era-specific block types
+-- TODO: The results of this type family should be ledger types,
```

Comment:
Can we avoid TODOs without a ticket attached ?

Reply:
Fair enough. I have created [ADP-3351](https://cardanofoundation.atlassian.net/browse/ADP-3351) to track this.